README.md

```diff
@@ -200,11 +200,30 @@ ResponseCache::forget('/some-uri');
 // Forget several URIs
 ResponseCache::forget(['/some-uri', '/other-uri']);
 
+// Using Tags (the cache driver you configured must supports tags)
+ResponseCache::forget(['/some-uri', '/other-uri'], ['tag1', 'tag2']);
+
 // Alternatively
 ResponseCache::forget('/some-uri', '/other-uri');
+
+
+// Advanced
+ResponseCache::cacheCleaner()
+    ->setMethod('PUT')
+    ->setHeaders(['foo'=>'bar'])
+    ->setCacheNameSuffix('100') 
+    //
+    //... any methods from Spatie\ResponseCache\CacheCleaner\AbstractRequestBuilder
+    //
+    ->forget('/some-uri');//same options as ResponseCache::forget()
+
 ```
 
-The `forget` method only works when you're not using a `cacheNameSuffix` in your cache profile.
+The `ResponseCache::forget` method only works when you're not using a `cacheNameSuffix` in your cache profile, 
+use `ResponseCache::cacheCleaner` to deal with `cacheNameSuffix`.
```

Comment:
Could you add an example of this?

Reply:
ok, i can add a more clear example in the readme.

`
ResponseCache::cacheCleaner()->setCacheNameSuffix('100') ->forget('/some-uri');
`

this will clear '/some-uri' with suffix "100" (so for user with id 100 by default)